src/app/views/query-response/queryResponse.styles.ts

```diff
@@ -9,6 +9,13 @@ export const queryResponseStyles = (theme: ITheme) => {
             backgroundColor: theme.palette.blue,
             borderRadius: '50%',
             display: 'inline-block'
+          },
+          labelStyles: {
```

Comment:
Awesome work abstracting this here. This could be more descriptive since it refers to labels that show empty state. Possibly `emptyStateLabel` would do?

Reply:
Right. That's more descriptive.